doc/source/ray-air/check-ingest.rst

```diff
@@ -105,10 +120,11 @@ Enabling Streaming Ingest
 
     Use streaming ingest when:
 
-     * you have large datasets that don't fit into memory;
-     * you want to process small chunks or blocks per window;
-     * you can use small windows with small data blocks minimizing or avoiding memory starvation or OOM errors; and
-     * your preprocessing step is not a bottleneck or not an expensive operation since it's re-executed on each pass over the data.
+    * you have large datasets that don't fit into memory; and
+    * reading the preprocessed dataset from disk is slower than re-executing the preprocessing step on each epoch
```

Comment:
I would reword this a bit:

`you have large datsets that don't fit into memory and avoiding disk spilling is more beneficial than caching a preprocessed dataset`.

I was a bit confused on whether the `reading` is talking about the initial dataset reading, or reading due to object spilling.

Reply:
Ah got it, thanks, will reword!